{"data": "Traceback (most recent call last):\n  File \"/Users/leoliu/My/proj/jiant/jiant/utils/zlog.py\", line 39, in log_context\n    yield self\n  File \"/Users/leoliu/My/proj/jiant/jiant/proj/main/runscript.py\", line 173, in run_loop\n    metarunner.run_train_loop()\n  File \"/Users/leoliu/My/proj/jiant/jiant/shared/metarunner.py\", line 51, in run_train_loop\n    self.eval_model()\n  File \"/Users/leoliu/My/proj/jiant/jiant/proj/main/metarunner.py\", line 162, in eval_model\n    self.eval_save()\n  File \"/Users/leoliu/My/proj/jiant/jiant/proj/main/metarunner.py\", line 224, in eval_save\n    use_subset=True,\n  File \"/Users/leoliu/My/proj/jiant/jiant/proj/main/runner.py\", line 145, in run_val\n    verbose=verbose,\n  File \"/Users/leoliu/My/proj/jiant/jiant/proj/main/runner.py\", line 306, in run_val\n    task=task, accumulator=eval_accumulator, labels=val_labels, tokenizer=tokenizer,\n  File \"/Users/leoliu/My/proj/jiant/jiant/tasks/evaluate/core.py\", line 533, in compute_metrics_from_accumulator\n    predictions_dict, metrics = self.compute_preds_and_metrics(task, accumulator)\n  File \"/Users/leoliu/My/proj/jiant/jiant/tasks/evaluate/core.py\", line 542, in compute_preds_and_metrics\n    preds = cls.get_preds_from_accumulator(task, accumulator)\n  File \"/Users/leoliu/My/proj/jiant/jiant/tasks/evaluate/core.py\", line 509, in get_preds_from_accumulator\n    guid_list = accumulator.get_guids()\n  File \"/Users/leoliu/My/proj/jiant/jiant/tasks/evaluate/core.py\", line 80, in get_guids\n    return np.concatenate(self.guid_list)\n  File \"<__array_function__ internals>\", line 6, in concatenate\nValueError: zero-dimensional arrays cannot be concatenated\n", "TIMESTAMP": 1605069411.156606}
